twentynineteen-blocks/style.css

```diff
@@ -99,6 +99,14 @@ Tags: one-column, flexible-header, accessibility-ready, custom-colors, custom-me
 
 /* Post title positioning */
 
+@media only screen and (min-width: 768px) {
+	.wp-site-blocks .site-content > .wp-block-group__inner-container > *:not(.align-full):not(.align-wide) {
```

Comment:
Not sure that is correct but I wanted to show all blocks in `site-content` as centered by default unless they have (align full/wide)

Reply:
Yeah, this is right, but it makes the h1 rule below it unnecessary. I'll combine them and push a new update. 👍